picard/ui/coverartbox.py

```diff
@@ -378,14 +378,23 @@ def load_remote_image(self, url, mime, data):
         except CoverArtImageError as e:
             log.warning("Can't load image: %s" % unicode(e))
             return
+
+        if config.setting["behaviour_on_image_drop"] == 'replace':
```

Comment:
Since this option only applies to `load_remote_image()` and isn't specific to drag'n'drop (well, it is at the moment, since `load_remote_image()` is only called when dropping, but we could imagine other ways to add a remote image), i think the option should be `load_remote_image_behavior` instead.